book/migrations/0001_create_base_structure/up.sql

```diff
@@ -66,3 +56,18 @@ CREATE TABLE books_authors (
     person_id INTEGER NOT NULL REFERENCES persons (id) ON UPDATE CASCADE ON DELETE RESTRICT,
     UNIQUE(book_id, person_id)
 );
+
+CREATE TABLE books_series (
+    id SERIAL PRIMARY KEY,
+    book_id INTEGER NOT NULL REFERENCES books ON UPDATE CASCADE ON DELETE CASCADE,
+    series_id INTEGER NOT NULL REFERENCES series ON UPDATE CASCADE ON DELETE CASCADE,
+    -- `book_id` set `UNIQUE`, cause we assume that one book will only be part of one series
+    UNIQUE(book_id)
```

Comment:
Could also go to the respective line, or was it intentional?

Reply:
I put it into a separate line, because
1. the line of book_id is already very full and
2. because I am not super sure if it is this condition is actually true forever and I want people to notice and think about it (although we also have the comment for that)